pandas/core/indexing.py

```diff
@@ -90,7 +90,7 @@ class IndexingMixin:
 
     @property
     def iloc(self) -> "_iLocIndexer":
-        """
+        f"""
```

Comment:
Thanks @simonjayhawkins! Good catch. I would need to figure out a way to avoid this f-string. 

I put f-string here is because the doc here use `{` and `}`, and we want to use this docstring latter in other functions.

Reply:
> Thanks @simonjayhawkins! Good catch. I would need to figure out a way to avoid this f-string.

I made [a small tweak in doc decorator](https://github.com/pandas-dev/pandas/pull/31970/commits/6cd80820e938a15dc5b1e76bb2f6bf079c47b0f4#diff-e41db70762f6c5b076f9b551651d4132), and it will fix the issue here.

Reply:
> I made [a small tweak in doc decorator](https://github.com/pandas-dev/pandas/pull/31970/commits/6cd80820e938a15dc5b1e76bb2f6bf079c47b0f4#diff-e41db70762f6c5b076f9b551651d4132), and it will fix the issue here.

ok